clang/tools/c-index-test/core_main.cpp

```diff
@@ -271,9 +271,10 @@ static void dumpModuleFileInputs(serialization::ModuleFile &Mod,
                                  ASTReader &Reader,
                                  raw_ostream &OS) {
   OS << "---- Module Inputs ----\n";
-  Reader.visitInputFiles(Mod, /*IncludeSystem=*/true, /*Complain=*/false,
-                        [&](const serialization::InputFile &IF, bool isSystem) {
-    OS << (isSystem ? "system" : "user") << " | ";
+  Reader.visitInputFiles(
+      Mod, /*IncludeSystem=*/true, /*Complain=*/false,
+      [&](const serialization::InputFile &IF, bool IsSystem, bool IsAffecting) {
+        OS << (IsSystem ? "system" : "user") << " | ";
     OS << IF.getFile()->getName() << '\n';
   });
```

Comment:
Indentation is wrong? Also should this print out IsAffecting?

Reply:
Ah, I'll reformat this before committing.

We can print that. I just didn't bother, since the ultimate goal is to get back to not serializing non-affecting module maps, which would increase churn in the tests and what not.

I can print it if you'd prefer.

Reply:
Ah, I don't think we need to if we're switching back soon.